### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by shikharcodes**
Added links to files `CONTRIBUTING.md` and `CODE_OF_CONDUCT.md` in `README.md` file.

## Fixes Issue

This PR fixes the following issues :
issue #7  

## Changes proposed

I've added links to files `CONTRIBUTING.md` and `CODE_OF_CONDUCT.md` in `README.md` file.

## Check List (Check all the boxes which are applicable)

- [x] Passing all the tests
- [x] Fixes the mentioned issue without conflicts
- [x] This changes needs documentation
- [x] I have made the required changes in documentation
- [x] This PR does not contain copyrighted content.

## Screenshots 
image

### CODE REVIEW

**Comment by shikharcodes** [comment]
I've made the requested additions. @kaiwalyakoparkar you can now merge this. :)

**Comment by kaiwalyakoparkar** [comment]
Hey @shikharcodes, this is so good. But, can you do one thing. Refer **" Note to new contributors "**  of https://github.com/kaiwalyakoparkar/NameIt. I would love more if you add it that way.

**Comment by shikharcodes** [comment]
> Hey @shikharcodes, this is so good. But, can you do one thing. Refer **" Note to new contributors "** of https://github.com/kaiwalyakopakar/NameIt. I would love more if you add it that way.

This link you shared is a broken link. 
Also, are you wanting something like this that when a person is about to make a pull request, a pop up should come up asking him/her to read the guidelines before contributing?

**Comment by kaiwalyakoparkar** [comment]
> > Hey @shikharcodes, this is so good. But, can you do one thing. Refer **" Note to new contributors "** of https://github.com/kaiwalyakopakar/NameIt. I would love more if you add it that way.
> 
> This link you shared is a broken link.
> Also, are you wanting something like this that when a person is about to make a pull request, a pop up should come up asking him/her to read the guidelines before contributing?

I have fixed the link now. You can now see it. Also, that's a nice idea for pop-up.

**Review by shikharcodes — COMMENTED** [review]
I've added the section as requested.

**Comment by shikharcodes** [comment]
It's good now you can merge it @kaiwalyakoparkar.

File: README.md [hunk 1]
```diff
@@ -13,6 +13,10 @@
 
 </div>
 
+## ❤️ Note for new Contributors : ❤️
+
+Thank you very much for considering this project. Make sure you read [Code of Conduct](https://github.com/kaiwalyakoparkar/flashtype/blob/master/CODE_OF_CONDUCT.md) and [Contribution Guidelines](https://github.com/kaiwalyakoparkar/flashtype/blob/master/CONTRIBUTING.md) before making contributions. Feel free to raise an issue and after gettting assigned to it, you are all set to work on it.
```
Comment (kaiwalyakoparkar) [inline README.md]: Instead of the entire link. You can add CODE_OF_CONDUCT.md as a link itself. like [ code of conduct (CODE_OF_CONDUCT.md)

**Review by kaiwalyakoparkar — CHANGES_REQUESTED** [review]

**Review by shikharcodes — COMMENTED** [review]
Done 👍🏻

**Review by kaiwalyakoparkar — APPROVED** [review]
Wow! You did a great job. Happy coding :+1:

**Pull request merged**